components/assetcard/index.css

```diff
@@ -136,89 +175,56 @@ governing permissions and limitations under the License.
   align-items: center;
 
   /* space between the asset and the header, place it on the header so header can be optional */
-  margin-block-start: var(--spectrum-global-dimension-static-size-200);
+  margin-block-start: var(--mod-assetcard-header-margin-block-start, var(--spectrum-assetcard-header-margin-block-start));
 }
 
 /* bold title */
 .spectrum-AssetCard-title {
```

Comment:
I took a different approach to setting up the typography here - https://github.com/adobe/spectrum-css/pull/2250/files#diff-9a31086a7f778cd08f7edbddfa8a949a9d8cf3871d6fe3d4eb63b5aa92de334cR63 - did you want to take a look and see if this would make sense to use? It allows for more typography cascading and contextual updating as well as including the language modifiers we might need.

Reply:
I added in the cjk custom properties (in looking at it noticed we do not have size tokens for cjk body text just cjk heading).

Could you explain more about what other changes you're suggesting?